apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -1,4 +1,5 @@
 import Octicons from "@expo/vector-icons/Octicons";
+import { countBy, isEqual } from "lodash";
```

Comment:
in my experience before `lodash` has been a very heavy dependency / high bundle size etc. Is that the case for `react-native` bundling too? if so might recommend against using it and pulling out the `statusCount` into its own file in `../logic/`

Reply:
Removed 🫡